substrate/bin/utils/chain-spec-builder/tests/test.rs

```diff
@@ -0,0 +1,186 @@
+// This file is part of Substrate.
+
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+use std::fs::File;
+
+use clap::Parser;
+use sc_chain_spec::update_code_in_json_chain_spec;
+use staging_chain_spec_builder::ChainSpecBuilder;
+
+const RUNTIME_PATH: &str =
+	"../../../../target/release/wbuild/substrate-test-runtime/substrate_test_runtime.wasm";
```

Comment:
Can we assume this is always present? When I just execute `cargo test` for this crate, the tests fail because this file is missing.

Reply:
My bad, I was not passing `--release`. Would have been nice to support debug tests too, but if its more as a hassle I think its okay.

Reply:
we could pass wasm blob to the run function. I'll improve it.